packages/react-magma-docs/src/pages/api/select.mdx

```diff
@@ -800,6 +801,82 @@ interface CustomSelectItem {
 Out of the box, the `Select` component uses React Magma-styled iconography for the button used to clear the selection and the caret for the dropdown.
 However, these can be overridden by providing custom components to the `components` prop, in an object containing a `ClearIndicator` and a `DropdownIndicator`.
 
+### Custom Item Component
+
+By default each item will be rendered with the `itemToString` value in a styled `li`. If you would like to add more information or
+would just like to be able to fully customize the look of your items you can pass in an `Item` component that uses the `props` that
```

Comment:
I want to make sure the props that are passed get documented. I looked at the generated `properties.json` and it didn't look right. I might not be understanding that output, so if it's working, great. I just want to make sure we're not leaving folks guessing.

Reply:
Right now select and combobox do not have generated props tables. Those will be added in a different PR. I will check the props tables built here to make sure it reflects what is there now.